test/scenarios.test.ts

```diff
@@ -34,26 +37,31 @@ describe('Testbot', () => {
 	it(
 		'can control DUT power',
 		async () => {
-			await testbotHat.setVout(deviceInteractor.powerVoltage);
-			await testbotHat.powerOnDUT();
+			// await testbotHat.setVout(deviceInteractor.powerVoltage);
+			// await testbotHat.powerOnDUT();
+			await deviceInteractor.powerOn();
```

Comment:
Wow, we weren't even using the tools we built. That's somewhat shocking. We created the createDeviceInteractor in `./tools` to handle this part only.

Reply:
@vipulgupta2048 not sure what you want to do here?

Reply:
My bad, no action required. I was really surprised we created the createDeviceInteractor only to not use it till now. Resolving